extensions/wikia/ArticleVideo/scripts/featured-video.jwplayer.ads.js

```diff
@@ -0,0 +1,95 @@
+define('wikia.articleVideo.featuredVideo.ads', [
```

Comment:
What do you think about renaming the module name to match file name?

Reply:
I wanted to keep X-Wing convention of all other files in this directory

Reply:
Makes sense